src/graphql/resolvers/mutations/adminSessionCreate.js

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable import/prefer-default-export */
+import debug from 'debug';
+
+import sessionStore from '../../../dataSources/cloudFirestore/session';
+import memberStore from '../../../dataSources/cloudFirestore/member';
+
+const dlog = debug('that:api:sessions:mutation:AdminSessionCreate');
+
+async function createNewSession(eventId, user, session, firestore) {
+  const [sessionResults, userResults] = await Promise.all([
+    sessionStore(firestore).adminCreate({
+      eventId,
+      session,
+    }),
+    memberStore(firestore).find(user.sub),
+  ]);
+
+  return { sessionResults, userResults };
+}
+
+function sendUserEvent(sessionResults, userResults, userEvents) {
+  userEvents.emit('newSessionCreatedByAdmin', {
```

Comment:
invalid event.. see other comment below.

Reply:
@csell5 again thinking we may want a different eventName for the admin actions to separate them from user events

Reply:
@csell5 replaced with defined event name. tested and email is sent on create/update